test/source/mock/google/exported-messages/message-export-183ec175f060b2c2.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
@rrrooommmaaa FYI Need to review https://github.com/FlowCrypt/flowcrypt-browser/pull/4742 first